src/Lounge.Web/Pages/Stats.cshtml

```diff
@@ -0,0 +1,640 @@
+            <dt>@Localizer["Median MMR"]</dt>
+            <dd id="median-mmr"></dd>
+        </div>
+    </dl>  
+</div>  
+
+<h1 class="m-4 text-center">Country Data</h1>
+
+<div class="row m-2">
+    <canvas id="statPopulationCountryChartBody" class="col-lg-6 col-md-6 col-sm-12 col-xs-12"></canvas>
+    <div class="table-responsive table-sm my-auto col-lg-6 col-md-6 col-sm-12 col-xs-12" >
+        <table id="country-population-table" class="table table-striped table-dark text-center">
+            <thead>
+                <tr>
+                    <th>@Localizer["Country"]</th>
+                    <th>@Localizer["Population"]</th>
+                    <th>@Localizer["% of Total Population"]</th>
+                </tr>
+            </thead>
+        </table>
+    </div>
+</div>
+
+<canvas class="m-2" id="statOverallCountryChartBody"></canvas>
+
+<canvas class="m-2" id="statTopCountryChartBody"></canvas>
+
+<h1 class="m-4 text-center">Activity Data</h1>
+
+<div class="row m-2">
+    <canvas id="statMogiFormatChartBody" class="col-lg-6 col-md-6 col-sm-6 col-xs-6"></canvas>
+    <div class="table-responsive my-auto col-lg-6 col-md-6 col-sm-6 col-xs-6" >
+        <table id="mogi-format-table" class="table table-striped table-dark text-center">
+            <thead>
+                <tr>
+                    <th>@Localizer["Format"]</th>
+                    <th>@Localizer["Mogis"]</th>
+                    <th>@Localizer["% of Mogis"]</th>
+                </tr>
+            </thead>
+        </table>
+    </div>
+</div>
+
+<canvas class="m-2" id="statMogiActivityChartBody"></canvas>
+
+<div class="row m-2">
+    <div class="table-responsive table-sm my-auto col-lg-6 col-md-6 col-sm-6 col-xs-12" >
+        <table id="mogi-tier-table" class="table table-striped table-dark text-center">
+            <thead>
+                <tr>
+                    <th>@Localizer["Tier"]</th>
+                    <th>@Localizer["Mogis"]</th>
+                    <th>@Localizer["% of Mogis"]</th>
+                </tr>
+            </thead>
+        </table>
+    </div>
+    <div class="table-responsive table-sm my-auto col-lg-6 col-md-6 col-sm-6 col-xs-12" >
+        <table id="mogi-weekday-table" class="table table-striped table-dark text-center">
+            <thead>
+                <tr>
+                    <th>@Localizer["Day of Week"]</th>
+                    <th>@Localizer["Mogis"]</th>
+                    <th>@Localizer["% of Mogis"]</th>
+                </tr>
+            </thead>
+        </table>
+    </div>
+</div>
+
+@section Scripts
+    {
+            <script src="~/lib/jquery/jquery.min.js"></script>
+            <script src="~/lib/chartjs/chart.min.js"></script>
+            <script src="~/js/stats.js"></script>
+            <script>
+                const countryNames = @Json.Serialize(LoungeSettingsService.CountryNames)
+                const season = document.getElementById("seasonInput").value;
```

Comment:
77ed48a2a9a577a44aff27b2bf6dfb086e53ba9e

Reply:
I left the fetch on the cshtml file and just pass in season to the functions that need it, because i was getting an error trying to use the season variable from another file.  It was saying I couldnt use the variable before it was instantiated even though it didnt look like that would be an issue, some dumb async problem.  This should be fine though.  

I also just recreated the countryNames object in my big js object of chartSeasonConfig, because i was already doing that anyways for other stuff